### PR TITLE
[StorageBundle] JS : add x-amz-acl header if using acl setting

### PR DESCRIPTION
**Pull request by alanpilloud**
I'm using DigitalOcean Spaces which requires that header to validate the signature. This might be required by other object storages services too.

This simply adds a "x-amz-acl" header containing the acl setting from the domain definition.

### CODE REVIEW

**Comment by franzwilding** [comment]
Nice work! @stefan-kamsker will have a look

**Comment by stefan-kamsker** [comment]
Hi @alanpilloud,

some tests are failing with this change, can you fix them?
fe.
https://travis-ci.org/unite-cms/unite-cms/jobs/544803783

Thanks!

BR

**Comment by alanpilloud** [comment]
Hi @stefan-kamsker 

How do you run tests ?

I'm using the docker configuration, running "./vendor/bin/phpunit" in the container doesn't show the same results that the ones on Travis.

**Comment by stefan-kamsker** [comment]
Hi, 

unfortunately there's a problem with running the tests inside docker image, i have to fix that

normally you would run it with 
./vendor/phpunit/phpunit/phpunit
./vendor/phpunit/phpunit/phpunit --configuration phpunit.xml etc (if the test database was created and the settings are correct)

BR

**Comment by alanpilloud** [comment]
I really don't understand what is going on.
I've cloned directly this repository and I have the same errors, but then I dont understand why some other builds have passed.

I decided to test this way to isolate the Storage Bundle : 
docker exec unitecms-web ./vendor/phpunit/phpunit/phpunit src/Bundle/StorageBundle/Tests

**Comment by alanpilloud** [comment]
@stefan-kamsker  @franzwilding  Could one of you try to reproduce my changes in your dev environment ? I tried again to solve this issue, but still I have no luck and no Idea why it would happen.

File: src/Bundle/StorageBundle/Field/Types/FileFieldType.php [hunk 1]
```diff
@@ -93,6 +93,7 @@ function getFormOptions(FieldableField $field): array
             'endpoint' => $this->generateEndpoint($field->getSettings()),
             'upload-sign-url' => $url,
             'upload-sign-csrf-token' => $this->csrfTokenManager->getToken('pre_sign_form'),
+            'acl' => $field->getSettings()->bucket['acl'],
```
Comment (stefan-kamsker) [inline src/Bundle/StorageBundle/Field/Types/FileFieldType.php]: acl bucket settting is not always there (fe. in the unit tests)

`'acl' => $field->getSettings()->bucket['acl'] ?? ''`

**Comment by alanpilloud** [comment]
@stefan-kamsker Great it was indeed this simple issue. Thanks a lot for your help !

**Pull request merged**